content/faq.md

```diff
@@ -9,17 +9,32 @@ hidetitle = "true"
 
 ### What is Urbit? {#what-is-urbit}
 
-Urbit reimagines the internet as a peer-to-peer network designed to stay that way. Previous attempts to "decentralize" the internet have only attempted to solve certain corners of the problem. Urbit proposes that only a new stack, built from the ground up as a tightly integrated system, can realize a great user experience for people using a peer-to-peer web.
-
-Combining a deterministic operating system (Urbit OS / Arvo) and a secure, global identity layer (Urbit ID / Azimuth), the Urbit project has created something that’s never existed before: an encrypted peer-to-peer network where you can be sure that messages are never tampered with or surveilled, and where strangers on that network can be trusted by default.
+Urbit reimagines the internet as a peer-to-peer network designed to stay that
+way. Previous attempts to "decentralize" the internet have only attempted to
+solve certain corners of the problem. Urbit proposes that only a new stack,
+built from the ground up as a tightly integrated system, can realize a great
+user experience for people using a peer-to-peer web.
+
+Combining a deterministic operating system ([Urbit OS](#what-is-arvo) / Arvo) and a secure,
+global identity layer ([Urbit ID](#what-is-urbit-id) / [Azimuth](#what-is-azimuth)), the Urbit project has created
+something that’s never existed before: an encrypted peer-to-peer network where
+you can be sure that messages are never tampered with or surveilled, and where
+strangers on that network can be trusted by default.
```

Comment:
> and where strangers on that network can be trusted by default.

I hate this. What does it even mean?

Reply:
I think a better way to put this is Sybil resistant https://github.com/urbit/urbit.org/pull/538/commits/de3929932db98ec64e903fc600d2f3cc7415644a